README.md

```diff
@@ -1,69 +1,45 @@
 # MCS Benchmark Data
 
-MCS Benchmark Data is a set of scripts that can be ued to create and execute Extract, Transform, Load (ETL) pipelines that store information about Machine Commonsense benchmarks into the Commonsense Benchmark Ontology.
+MCS Benchmark Data is a set of scripts that can be used to create and execute pipelines that extract, transform and load information about Machine Commonsense benchmarks into the Commonsense Benchmark Ontology.
 
 For more information about the ontology, refer to:
 [Commonsense Benchmark Ontology](https://tetherless-world.github.io/mcs-ontology/docs/index-en.html)
 *Henrique O. Santos, Minor Gordon, Zhicheng Liang, Gretchen Forbush, Deborah L. McGuinness*
+## One-time Setup
 
+### Create the Python virtual environment
 
-## Usage: Adding a Pipeline
+From the current directory:
+    python3 -m venv venv

```

Comment:
Make sure you preview the README before committing it. There should be an empty line above this one.